great_expectations/rule_based_profiler/data_assistant/data_assistant_dispatcher.py

```diff
@@ -74,29 +78,34 @@ def register_data_assistant(
             data_assistant: "DataAssistant" class to be registered
         """
         data_assistant_type = data_assistant.data_assistant_type
-        registered_data_assistants: Dict[
-            str, Type[DataAssistant]
-        ] = DataAssistantDispatcher._registered_data_assistants
-
-        if data_assistant_type in registered_data_assistants:
-            if registered_data_assistants[data_assistant_type] == data_assistant:
-                logger.info(
-                    f'Multiple declarations of DataAssistant "{data_assistant_type}" found.'
-                )
+        cls._register(data_assistant_type, data_assistant)
+
+        alias: Optional[str] = data_assistant.__alias__
+        if alias is not None:
+            cls._register(alias, data_assistant)
+
+    @classmethod
+    def _register(cls, name: str, data_assistant: Type[DataAssistant]) -> None:
+        registered_data_assistants = cls._registered_data_assistants
```

Comment:
We should just blow up here if we register two assistants with the same name. Throw a ValueError

Reply:
I agree - I've changed this to reflect this.